xml/schema/rolesAndUserGroups.xml

```diff
@@ -40,6 +40,10 @@
 			<NOTNULL/>
 			<DEFAULT VALUE="0"/>
 		</field>
+		<field name="permit_metadata_edit" type="I1">
+			<NOTNULL/>
+			<DEFAULT VALUE="0"/>
```

Comment:
Have you tested this for upgrades? I think ADODB might fail to add the default, making the `NOT NULL` addition cause an error.

Reply:
@asmecher I did test it running both the installation process and the upgrade process, without any issues. Do you think that I should check something else regarding that?

Reply:
I suspect it'll crop up with PostgreSQL if anywhere; leave it for now and I'll test that before release.

Reply:
I am relying on the Travis tests for PostgreSQL. Don't have it installed on my dev machine. Could install it if necessary.